src/test/integration/suite/environments.test.it.ts

```diff
@@ -0,0 +1,94 @@
+import { mockGetEnvironment, mockGetOrganization } from "../mocks/server";
+import { mockGitRepoAndBranch } from "../mocks/git";
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
also here, why `ts-ignore`? let's avoid those comments if it's possible. ( eslint rule? )

Reply:
https://github.com/env0/vscode-env0/pull/44